server/types.graphql

```diff
@@ -17,6 +17,7 @@ type User @model {
   flowdockName: String
   description: String
   role: UserRole @defaultValue(value: REGULAR)
+  projects: Project @relation(name: "ProjectChampions")
```

Comment:
We want a many-to-many relationship here since members can have multiple projects they are championing/have championed. I think we might also want to have this called `projectsChampioned` or something more specific since we could have other relationships to projects from users as well.